crates/primitives/src/types/transaction.rs

```diff
@@ -89,7 +86,7 @@ pub struct Transaction {
     )]
     #[cfg_attr(
         feature = "serde",
-        serde(flatten, skip_serializing_if = "Option::is_none")
+        serde(flatten)
```

Comment:
double check this "flatten".  i think it should be removed.

Reply:
ok, checked, no other flatten or skip